spec/FlexibleHttpClientSpec.php

```diff
@@ -92,3 +88,22 @@ function it_does_not_emulate_a_client(FlexibleHttpClient $client, RequestInterfa
         $this->sendAsyncRequest($asyncRequest);
     }
 }
+
+class CombinedHttpClient implements HttpClient, HttpAsyncClient
+{
+    /**
+     * @param RequestInterface $request
+     */
+    public function sendRequest(RequestInterface $request)
+    {
+        // TODO: Implement sendRequest() method.
```

Comment:
It's a stub. We need something which implements both interfaces, and it can't be final.